wrappingGsdk/fakegame/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace fakegame
+{
+    class Program
+    {
+        async static Task Main(string[] args)
+        {
```

Comment:
maybe we should have a more interesting game one that at least responds to a request like WinRunner